app/helpers/application_helper.rb

```diff
@@ -80,6 +80,7 @@ def revision_path(rev)
   end
 
   def link_to_revision(rev)
+    logger.error('The rails helper "link_to_revision" is deprecated for this application. To ensure your revision links are able to show the user-friendly label, use the _link.slim partial instead.')
```

Comment:
Not error, warn. and it must be called as Rails.logger.warn
`Rails.logger.warn("error message")`

Reply:
@mcrutch, done. Sorry, got mixed up with Django for the logger part.